core/api/test/integration/app/user/consent-list.spec.ts

```diff
@@ -0,0 +1,102 @@
+import { exec } from "child_process"
+import puppeteer from "puppeteer"
+
+import { Admin } from "@/app"
+import { consentList } from "@/services/hydra"
+import { sleep } from "@/utils"
+
+let userId: UserId
+const email = "test@galoy.io" as EmailAddress
+
+beforeAll(async () => {
+  const account = await Admin.getAccountByUserEmail(email)
+  if (account instanceof Error) throw account
+
+  userId = account.kratosUserId
+})
+
+const getOTP = (email: string): Promise<string> => {
+  return new Promise((resolve, reject) => {
+    const query = `docker exec -i galoy-dev-kratos-pg-1 psql -U dbuser -d default -t -c "SELECT body FROM courier_messages WHERE recipient='${email}' ORDER BY created_at DESC LIMIT 1;"`
```

Comment:
not sure about this.. is not better to use knex? we already have the package.. wdyt @vindard

Reply:
that's fair. I copied it from the dashboard.